library/CoreBundle/Resources/config/parameters.yml

```diff
@@ -9,6 +9,13 @@ parameters:
     database_name: ivozprovider
     database_user: root
     database_password: changeme
+    api_platform_base_url: 'https://localhost/api/platform/dev.php/'
```

Comment:
Is this _/dev.php/_ path intentional here?

Reply:
Umm, kind of... that's the endpoint for `Ivoz\Core\Infrastructure\Service\Rest\Client`, `dev.php` makes things easier in development